azurelinuxagent/common/osutil/default.py

```diff
@@ -71,40 +71,47 @@ def _add_wait(wait, command):
     return command
 
 
-def _get_iptables_version_command():
+def get_iptables_version_command():
     return ["iptables", "--version"]
 
+def get_firewall_accept_dns_tcp_rule(wait, command, destination):
```

Comment:
Why do we need this function at all? Why can't we directly just use `AddFirewallRules.get_firewall_accept_dns_tcp_rule` instead?

Reply:
As discussed offline all the methods in default.py will be moved to networkutil